content/docs/gke/gcp-e2e.md

```diff
@@ -72,7 +73,8 @@ In summary:
 * Setting up [Kubeflow][kubeflow] in a [GKE][kubernetes-engine]
   cluster.
 
-* Testing the code locally using a [Jupyter notebook][jupyter-notebook].
+* Testing the code locally using a 
```

Comment:
Should we delete this? I don't think running the code locally anymore applies. The notebook is designed to run in the cluster not their local machine. They would probably have problems running it locally

Reply:
Good idea. Done.